include/seqan3/alphabet/composite/alphabet_tuple_base.hpp

```diff
@@ -199,67 +199,67 @@ class alphabet_tuple_base :
             return get<index>(lhs) == static_cast<alphabet_type>(rhs);
         }
 
-        //!\copydoc operator==(derived_type const lhs, component_proxy const rhs)
+        //!\copydoc seqan3::alphabet_tuple_base::component_proxy::operator==(derived_type const,component_proxy const)
```

Comment:
Our code guidelines require space after comma... (+10x below)
```suggestion
        //!\copydoc seqan3::alphabet_tuple_base::component_proxy::operator==(derived_type const, component_proxy const)
```